src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx

```diff
@@ -110,20 +111,13 @@ const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playl
     setActiveChannel(channelId);
   };
 
-  // Effects
```

Comment:
We might want to leave this. I'm not sure if there's any EPG auto-advance stuff that may break without it or if there are other incoming links that rely on it.